frontend/cypress/integration/featureFiles/istio_config.feature

```diff
@@ -9,50 +10,276 @@ Feature: Kiali Istio Config page
+    And the http-route of the VirtualService also has a destination to host "sleep" and subset "v1" with weight 50
+    And a "foo" DestinationRule in the "default" namespace for "sleep" host
+    And the DestinationRule has a "v1" subset for "version=v1" labels
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+
+  @crd-validation
+  Scenario: KIA1106 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep"
+    And the VirtualService applies to "sleep" hosts
+    Given there is a "bar" VirtualService in the "default" namespace with a "bar-route" http-route to host "sleep"
+    And the VirtualService applies to "sleep" hosts
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+    And the "bar" "VirtualService" of the "default" namespace should have a "warning"
+
+  @crd-validation
+  Scenario: KIA1107 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep" and subset "v1"
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+
+# TODO: KIA06xx and KIA07xx does not appear in Istio Config list page. They appear in Svc/workload lists.
+#   Thus, these validations do not belong to this feature file.
+
+# TODO: KIA0801 is only applicable for Maistra. We don't have an environment to run tests for this one.
+
+# TODO: KIA090x no longer apply to current Istio (ServiceRole does not exist anymore)
+#   Even more, the checkers are no longer in the codebase. The docs should be updated.
+
+# TODO: Both KIA1003 and KIA1103 are stale validations. Istio rejects a resource with such configs (i.e. it
+#   is not accepted into the cluster).
+#   Below is a scenario that was prepared to test this checker, but it is "red" because when trying
+#   to create an offending resource, the `kubectl apply` command terminates with error because of the offending configs.
+#
+#  @crd-validation
+#  Scenario: KIA1003 validation
+#    Given there is a "foo" Sidecar resource in the "default" namespace that captures egress traffic for hosts "noslashsymbolpresent"
+#    And the Sidecar is applied to workloads with "app=foo" labels
+#    When user selects the "default" namespace
+#    Then the "foo" "Sidecar" of the "default" namespace should have a "danger"
+
+# TODO: Apparently, Kiali does not trigger:
+#   KIA0204, KIA0205, KIA0206, KIA0401, KIA0501
+#   It is possible that under the current mTLS defaults these
+#   validations may became obsolete and may never happen anymore.
+#   Below, there are some Scenarios that were prepared to teset some of these chekers,
+#   but they are "red", because of the non-triggering validation.
+#   Also, KIA1108 is not triggering for some unknown reason.
```

Comment:
@hhovsepy As I briefly talked with Lucas via chat, I wonder if you could give some help to check if the mentioned validations in this comment are still effective, or if they have became dead validations.